docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+different variables, then making further predictions based on these 
+relationships.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. Our imaginary dataset consists of 
+three data points:
+
+|===
+| Size (m2) | Monthly rent 
+| 44        | 1600
+| 24        | 1055
+| 63        | 2300
+|===
+
+After the model determines the relationship between the {feature-vars} 
+(apartment size) and the {depvar} (rent), it can make predictions such as the 
+monthly rent of a hundred square meter-size apartment.
+
+This is a simple example. Usually {regression} problems are multi-dimensional, 
+so the relationships that {reganalysis} tries to find are between multiple 
+{feature-vars} and the {depvar}. To extend our example, a more complex 
+{reganalysis} could take into account further {feature-vars} like the location 
+of the apartment in the city, on which floor it is, and whether the apartment 
+has a riverside view or not and so on.
+
+
+[discrete]
+[[dfa-regression-features]]
+=== {feature-vars-cap}
+
```

Comment:
If users really need to understand the concept of feature variables and dependent variables, I think this is the place to do it. Maybe something like this:
```suggestion

When you perform {reganalysis}, you must identify a subset of fields that you want to use to create a model for predicting other fields. We refer to these fields as _feature variables_ and _dependent variables_, respectively.
```